tests/test_request_attrs.py

```diff
@@ -5,14 +5,31 @@
 
 import falcon
 from falcon.request import Request, RequestOptions
-from falcon.request_helpers import make_etag
 import falcon.testing as testing
 import falcon.uri
 from falcon.util import compat
+from falcon.util.structures import ETag
 
 _PROTOCOLS = ['HTTP/1.0', 'HTTP/1.1']
 
 
+def _make_etag(value, is_weak=False):
```

Comment:
Moved this in here because it was still useful for constructing test data, if nothing else.